samples/net/mqtt/src/modules/transport/Kconfig.transport

```diff
@@ -34,6 +34,19 @@ config MQTT_SAMPLE_TRANSPORT_BROKER_HOSTNAME
 	string "MQTT broker hostname"
 	default "test.mosquitto.org"
 
+config MQTT_SAMPLE_TRANSPORT_USER_NAME
+	string "MQTT client user name"
+	default ""
+	help
+	  username used for password based authentication
+
+config MQTT_SAMPLE_TRANSPORT_PASSWORD
+	string "MQTT Client password"
+	default ""
+	help
+	  set password for password based authentication. If password is provided then 
```

Comment:
Please start the sentence with capital letter.

Reply:
fixed in https://github.com/nrfconnect/sdk-nrf/pull/12430/commits/fa079d64e6fbb8c98943ff6bf9c5ffb23bed2079